.github/workflows/publish-gihub-release.yml

```diff
@@ -0,0 +1,28 @@
+---
+on:
+  push:
+    tags:
+    - v[0-9]+.[0-9]+.[0-9]+*
+
+jobs:
+  test:
+    uses: ./.github/workflows/.test.yml
+  push-docker:
+    uses: ./.github/workflows/.docker-push.yml
+    with:
+      is_pr: false
+    secrets:
+      dockerhub_username: "${{ secrets.DOCKERHUB_USERNAME }}"
+      dockerhub_token: "${{ secrets.DOCKERHUB_TOKEN }}"
```

Comment:
You can use the [inherit secrets](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsecretsinherit) option and ditch this input for simplicity  .

Reply:
I though that having `.docker-push.yml` be explicit rather than implicit in which variables it needs would simplify its maintenance (e.g. better error messages, easier future refactoring, etc).

But then again it's a deeply-coupled reusable workflow, which will live and die in this repo, so i'd be ok also to go for secret inheritance. Wdyt?

Reply:
Yeah exactly . That was my thought . This is not a widely reusable workflow . We have https://github.com/mattermost/actions and https://github.com/mattermost/actions-workflows repo for our reusable ones

Reply:
Ok, done. The [test run in the fork](https://github.com/mvitale1989/elrond/actions/runs/4054318673/jobs/6976055561) confirms that the secrets now get correctly inherited